tests/types/tisop.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-  disabled: true
+    disabled: true
```

Comment:
Disabled around 2014 in a commit that 'tuple field names are ignored'

Reply:
Might already be in the spec, and what isn't can either be added or discarded.